File: Modul3/Python/PRAK305-2310817310012-MUHAMMAD_AZWIN_HAKIM.py
i = 1

while i <= 5:
    d = int(input())

    jam = d // 3600
    sisadetik = d % 3600

    menit = sisadetik // 60
    detik = sisadetik % 60

    if 0 < jam <= 24:
        print(f"{jam:02d}:{menit:02d}:{detik:02d}")
    elif jam == 0 and menit > 0:
        print(f"{jam:02d}:{menit:02d}:{detik:02d}")
    elif jam > 24:
        hari = jam // 24
        jam = jam % 24
        print(f"{hari} hari {jam:02d}:{menit:02d}:{detik:02d}")

    i += 1

File: Modul3/C/PRAK305-2310817310012-MUHAMMAD_AZWIN_HAKIM.c
#include <stdio.h>

int main(){
    int d, jam, menit, sisadetik;

    int i = 1;

    while(i <= 5){
        (scanf("%d", &d));
        
        int jam = d / 3600;
        int sisadetik = d % 3600;
        
        int menit = sisadetik / 60;
        int detik = sisadetik % 60;
        {
            if (24 >= jam > 0){
                printf("%02d:%02d:%02d\n", jam, menit, detik);
                i++;
            }else if(jam == 0 && menit > 0){
                printf("%02d:%02d:%02d\n", jam, menit, detik);
                i++;
            }else if(jam > 24){
                printf("%d hari %02d:%02d:%02d\n", jam / 24, jam % 24, menit, detik);
                i++;
            }
        }
    }
}